### PR TITLE
Fixes the use of notebook argument for plot commands 

### PR DESCRIPTION
**Pull request by PProfizi**
When in a Jupyter notebook, the notebook=False argument was not working when using a plot command going through Plotter.plot_contour

### CODE REVIEW

File: ansys/dpf/core/plotter.py [hunk 1]
```diff
@@ -770,7 +749,7 @@ def plot_contour(
             version_to_reach = '0.32.0'
```
Comment (anslpa) [inline ansys/dpf/core/plotter.py:749]: Hi @PProfizi, could you please check that it does not break back the following : https://github.com/pyansys/pydpf-core/issues/140 ?

File: ansys/dpf/core/plotter.py [hunk 2]
```diff
@@ -601,10 +601,7 @@ def plot_chart(fields_container, off_screen=False, screenshot=None):
     def plot_contour(
             self,
             field_or_fields_container,
-            notebook=None,
             shell_layers=None,
-            off_screen=None,
-            show_axes=True,
```
Comment (anslpa) [inline ansys/dpf/core/plotter.py:607]: @PProfizi Would it be possible to check that the show_axes (not used in any examples I think) is still working ?

**Review by anslpa — APPROVED** [review]

File: ansys/dpf/core/plotter.py (hunk 2 above)
Comment (PProfizi) [reply ansys/dpf/core/plotter.py:607]: It is in the new example where all previous arguments are shown explicitly, with their default PyVista values.

File: ansys/dpf/core/plotter.py (hunk 1 above)
Comment (PProfizi) [reply ansys/dpf/core/plotter.py:749]: @anslpa I did, I found a bug, I fixed it, it should work :)

**Pull request merged**